CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 * EXODistributionGroup
   * Fixes the export of group membership to use Identity.
+  * Add support for the new acceptable value for UseNewTeamsClient ( NewTeamsAsDefault )
```

Comment:
Please validate the indentation and position of this entry

Reply:
Pushed a new version.
Expecting everything is now ok